sdk/src/cameras/ad-96tof1-ebz/calibration_96tof1.cpp

```diff
@@ -490,7 +497,9 @@ void Calibration96Tof1::buildGeometryCalibrationCache(
             double tanYAngle = (y0 - i) / fy;
```

Comment:
Division by 0 will continue to exist. Would be nice to avoid that as well.

Reply:
fixed